src/scripts/crowdin/import/utils.ts

```diff
@@ -0,0 +1,170 @@
+import { copyFileSync, existsSync, mkdirSync, readdirSync } from "fs"
+import { join } from "path"
+
+import i18Config from "../../../../i18n.config.json"
+import { INTL_JSON_DIR, TRANSLATIONS_DIR } from "../../../lib/constants"
+import { DOT_CROWDIN } from "../translations/constants"
+
+import { BucketsList, SelectionItem, SummaryItem, TrackerObject } from "./types"
+
+/**
+ * Some language codes used in the repo differ from those used by Crowdin.
+ * This is used to convert any codes that may differ when performing folder lookup.
+ */
+export const getCrowdinCode = (code: string): string =>
+  i18Config.filter((lang) => lang.code === code)?.[0]?.crowdinCode || code
+
+/**
+ * Reads `ls` file contents of `_path`, moving .md and .json files
+ * to their corresponding destinations in the repo. Function is called
+ * again recursively for subdirectories.
+ *
+ * @param path An absolute path to the directory being scraped.
+ * @param contentSubpath The subpath deep to the lang-code directory,
+ * used to construct destination for markdown content files
+ * @param repoLangCode Language code used within the repo
+ * @returns void
+ */
+export const scrapeDirectory = (
+  path: string,
+  contentSubpath: string,
+  repoLangCode: string,
+  trackers: TrackerObject
+): void => {
+  if (!existsSync(path)) return
+  const ls: string[] = readdirSync(path).filter(
+    (dir: string) => !dir.startsWith(".")
+  )
+  ls.forEach((item: string) => {
+    const source: string = join(path, item)
+    if (item.endsWith(".json")) {
+      const jsonDestDirPath: string = join(INTL_JSON_DIR, repoLangCode)
+      if (!existsSync(jsonDestDirPath))
+        mkdirSync(jsonDestDirPath, { recursive: true })
+      const jsonDestinationPath: string = join(jsonDestDirPath, item)
+      copyFileSync(source, jsonDestinationPath)
+      // Update .json tracker
+      trackers.langs[repoLangCode].jsonCopyCount++
+    } else if (
+      item.endsWith(".md") ||
+      item.endsWith(".svg") ||
+      item.endsWith(".xlsx")
```

Comment:
```suggestion
      item.endsWith(".md") 
```

I don't think we want to commit .svg and .xlsx file formats. There is a different process for  importing images that @nloureiro and @lukassim have sorted out I believe.